src/modules/paymentProcessor/PP_Simple_v1.sol

```diff
@@ -184,6 +184,17 @@ contract PP_Simple_v1 is Module_v1, IPaymentProcessor_v1 {
         _claimPreviouslyUnclaimable(client, token, receiver);
     }
 
+    /// @inheritdoc IPaymentProcessor_v1
+    function validPaymentOrder(
+        IERC20PaymentClientBase_v1.PaymentOrder memory order
+    ) external view returns (bool valid) {
```

Comment:
You are initializing a variable `valid` here via the return statement, which we usually only do if we actually use it. The other function you added doesn't do this, so just to be consistent I think this should just be `returns (bool)`?